Alien invasion/aliens.py

```diff
@@ -0,0 +1,144 @@
+
+class Aliens:
+    def __init__(self):
+        self.image1 = pygame.image.load("alien/better_ship.png")
+        self.width = 80
+        self.height = 80
+        self.speed1 = 2
+        self.x1 = 150
+        self.y1 = 10
+        self.scale_image1 = pygame.transform.scale(self.image1, (self.width, self.height))
+
+        self.screen = Screen()
+
+        self.image2 = pygame.image.load("alien/orange_ship.png")
+        self.width = 80
+        self.height = 80
+        self.speed2 = 2
+        self.x2 = 150
+        self.y2 = 100
+        self.scale_image2 = pygame.transform.scale(self.image2, (self.width, self.height))
+
+        self.image3 = pygame.image.load("alien/yellow_ship.png")
+        self.width = 80
+        self.height = 80
+        self.speed3 = 2
+        self.x3 = 150
+        self.y3 = 190
+        self.scale_image3 = pygame.transform.scale(self.image3, (self.width, self.height))
+
+        self.alien_army = []
+
+
+
+        self.move = {
+            "left":True,
+            "right":False
+        }
+
+    def spawn_first_wave(self):
+        for _ in range(15):
+            self.x1 += 90
+            self.x1 = self.x1
+            self.alien_army.append({
+                "alien":self.scale_image1,
+                "x":self.x1,
+                "y":self.y1,
+                "speed":self.speed1,
+                "alive":True
+            })
+
+        for _ in range(15):
+            self.x2 += 90
+            self.x2 = self.x2
+            self.alien_army.append({
+                "alien":self.scale_image2,
+                "x":self.x2,
+                "y":self.y2,
+                "speed":self.speed2,
+                "alive":True
+            })
+
+        for _ in range(15):
+            self.x3 += 90
+            self.x3 = self.x3
+            self.alien_army.append({
+                "alien":self.scale_image3,
+                "x":self.x3,
+                "y":self.y3,
+                "speed":self.speed3,
+                "alive":True
+            })
+
+    def spawn_second_wave(self):
+        for _ in range(15):
+            self.x1 += 90
+            self.x1 = self.x1
+            self.alien_army.append({
+                "alien":self.scale_image1,
+                "x":self.x1,
+                "y":self.y1,
+                "speed":self.speed1,
+                "alive":True
+            })
+
+        for _ in range(15):
+            self.x2 += 90
+            self.x2 = self.x2
+            self.alien_army.append({
+                "alien":self.scale_image2,
+                "x":self.x2,
+                "y":self.y2,
+                "speed":self.speed2,
+                "alive":True
+            })
+
+        for _ in range(15):
+            self.x3 += 90
+            self.x3 = self.x3
+            self.alien_army.append({
+                "alien":self.scale_image3,
+                "x":self.x3,
+                "y":self.y3,
+                "speed":self.speed3,
+                "alive":True
```

Comment:
Ten słownik się powtarza cały czas. Stwórz więc z tego klasę z tymi parametrami. Będzie łatwiej tym zarządzać i unikniesz błędów związanych z niepoprawnym nazwaniem kluczy

Reply:
To zostało